pontoon/base/middleware.py

```diff
@@ -48,3 +48,20 @@ def process_request(self, request):
             if user:
                 request.user = user
                 auth.login(request, user)
+
+
+class CookiesSameSite(object):
+    """
+    Support for SameSite attribute in Cookies is implemented in Django 2.0 and won't
+    be backported to Django 1.11.x.
+    This middleware will be obsolete when Pontoon will be ported to Django 2.0 and
+    should be removed as a follow-up action.
+
+    Related bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1461630
+    """
+    def process_response(self, request, response):
+        for cookie in ('sessionid', 'csrftoken'):
+            if cookie in response.cookies:
+                response.cookies[cookie]._reserved['samesite'] = 'SameSite'
+                response.cookies[cookie]['samesite'] = 'strict'
```

Comment:
'strict' means that regular links to the application also wont get the session cookie added, which is good for security but not so good for usability. 'lax' is sometimes a better compromise as it still prevents the more common attacks. If 'strick' works for pontoon then great, but being able to choose between them would be even better for reuse.